### ISSUE TITLE
[1.19] All players can not join the server after upgrading Forge and relevant mods

### ISSUE THREAD

**Issue by jellybean13** [henkelmax/simple-voice-chat#429]
**Bug description**
41.0.63: Everything works well.
41.0.99: Players instantly get disconnected after joining the world. However, this issue won't occur if this mod is removed from both server and client.

**Steps to reproduce the issue**
1. Install this mod with Forge 41.0.94 compatible on clients and the dedicated server.
2. Start dedicated server.
3. Join the dedicated server.
4. See error on client side.

**Expected behavior**
Players can join the world without errors.

**Log files**
Client: https://pastebin.com/Sa96vtaF
Server: https://pastebin.com/UcF3EzGA

**Versions**
- Minecraft version: 1.19
- Forge version: 41.0.99
- Mod version: 2.2.52

**Other mods**
```
[13:07:12] [Server thread/INFO] [minecraft/MinecraftServer]: Mod List: 
• minecraft server-1.19-20220607.102129-srg.jar : minecraft (1.19) - 1
• mods_folder SereneSeasons-1.19-8.0.0.19.jar : sereneseasons (0.0NONE) - 1
• mods_folder configured-1.5.5-1.19.jar : configured (1.5.5) - 1
• mods_folder Dynmap-3.4-beta-4-forge-1.19.jar : dynmap (3.4-beta-4-Dev) - 1
• mods_folder voicechat-forge-1.19-2.2.52.jar : voicechat (1.19-2.2.52) - 1
• mods_folder tombstone-8.0.3-1.19.jar : tombstone (8.0.3) - 1
• mods_folder open-parties-and-claims-forge-1.19-0.6.1.jar : openpartiesandclaims (0.6.1) - 1
• mods_folder Jade-1.19-forge-7.6.0.jar : jade (7.6.0) - 1
• mods_folder starlight-1.1.1+forge.cf5b10b.jar : starlight (1.1.1+forge.a3aea74) - 1
• mods_folder EditSign-1.19-2.4.2.jar : editsign (2.4.2) - 1
• mods_folder AI-Improvements-1.19-0.5.1.jar : aiimprovements (0.5.1) - 1
• mods_folder catalogue-1.6.3-1.19.jar : catalogue (1.6.3) - 1
• mods_folder jei-1.19-forge-11.0.0.222.jar : jei (11.0.0.222) - 1
• mods_folder dynview-1.19-2.8.jar : dynview (2.3) - 1
• minecraft forge-1.19-41.0.99-universal.jar : forge (41.0.99) - 1
• mods_folder toms_storage-1.19-1.3.8.jar : toms_storage (1.3.8) - 1
• mods_folder Xaeros_Minimap_22.12.0_Forge_1.19.jar : xaerominimap (22.12.0) - 1
• mods_folder XaerosWorldMap_1.26.1_Forge_1.19.jar : xaeroworldmap (1.26.1) - 1
• mods_folder MoreMobGriefingOptions-1.19-2.0.1.jar : moremobgriefingoptions (2.0.1) - 1
• mods_folder alternate-current-mc1.19-1.4.0.jar : alternate_current (1.4.0) - 1
```

**Screenshots (Optional)**
![image](https://user-images.githubusercontent.com/46275808/179384667-72d11244-c710-46ba-ae49-663fc75dac82.png)

**Comment by henkelmax** [comment]
Can you please send me the **full** logs of client and server?

**Comment by jellybean13** [comment]
Client-full: https://pastebin.com/q4rDmf55
Server-full: https://pastebin.com/Wf9273zg

**Comment by henkelmax** [comment]
This seems to be a Forge issue in newer versions.

**Comment by henkelmax** [comment]
Alright it seems to be a Forge bug that should be fixed in 1.19.1

https://github.com/MinecraftForge/MinecraftForge/issues/8883#issuecomment-1186521416
